assets/component-image-with-text.css

```diff
@@ -82,13 +82,92 @@
   align-items: flex-start;
   height: 100%;
   justify-content: center;
+  align-self: center;
   padding: 4rem calc(4rem / var(--font-body-scale)) 5rem;
 }
 
+.image-with-text__content--mobile-right > * {
+  align-self: flex-end;
+  text-align: right;
+}
+
+.image-with-text__content--mobile-center > * {
+  align-self: center;
+  text-align: center;
+}
+
+.image-with-text--overlap .image-with-text__content {
+  transform: translate(0 , -3rem);
+  width: 90%;
+  margin: 0 auto; 
+}
+
 @media screen and (min-width: 750px) {
   .image-with-text__grid--reverse .image-with-text__content {
     margin-left: auto;
   }
+
+  .image-with-text__content--bottom {
+    justify-content: flex-end;
+    align-self: flex-end;
+  }
+
+  .image-with-text__content--top {
+    justify-content: flex-start;
+    align-self: flex-start;
+  }
+
+  .image-with-text__content--desktop-right > * {
+    align-self: flex-end;
+    text-align: right; 
+  }
+
+  .image-with-text__content--desktop-left > * {
+    align-self: flex-start;
+    text-align: left;
+  }
+
+  .image-with-text__content--desktop-center > * {
+    align-self: center;
+    text-align: center;
+  }
+
+  .image-with-text--overlap .image-with-text__text-item {
+    display: flex;
+    padding: 3rem 0; 
+  }
+  
+  .image-with-text--overlap .image-with-text__content {
+    height: auto;
+    min-width: calc(100% + 4rem);
+    max-width: calc(100% + 4rem);
```

Comment:
No. Because the width changes as you resize the page. This ensures that the position of the text box is always the same relative to the image, and always takes up 100% of the space available.

Reply:
No. Just assigning `width` doesn't force it to be the full width. but I can use width/min-width instead.